.babelrc

```diff
@@ -1,4 +1,7 @@
 {
   "presets": ["es2015"],
-  "plugins": ["transform-decorators-legacy"]
+  "plugins": [
+    "transform-decorators-legacy",
+    "add-module-exports"
```

Comment:
yeah basically `module.exports = exports['default'];` that was removed in babel 6